test/e2e/playbook/provision.yaml

```diff
@@ -85,7 +130,6 @@
           - kubeadm
           - kubectl
           - systemd-container
-          - helm
```

Comment:
I think this breaks our tests (on the default fedora VM). It stops installing Helm explicitly, so unless something pulls it in as a dependency (which I think won't happen) we'll end up without helm in the VM.

Could you update this PR to add the helm repo to the sources.list.d and then pull it in among the common packages ? We're fine with doing that on fedora test VMs, so it should be fine on Ubuntu as well.

Here are the [necessary details](https://helm.sh/docs/intro/install/#from-apt-debianubuntu) for doing it.

Reply:
Ok, so the changes unconditionally install from release tarballs. So I think this should be fine after all.